windows/d2-installer.nsi

```diff
@@ -355,10 +367,13 @@ Function DetectVSAndSDK
     StrCpy $VCPath $0
     StrCpy $VCVer $1
 
-    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot81"
+    ReadRegStr $0 HKLM "Software\Microsoft\Microsoft SDKs\Windows\v10.0" "InstallationFolder"
+    IfErrors 0 done_sdk
+    ClearErrors
+    ReadRegStr $0 HKLM "Software\Microsoft\Microsoft SDKs\Windows\v8.1" "InstallationFolder"
```

Comment:
I only have `v8.1a` in my registry and there is no `Lib` folder under `C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1` or `\v8.1a`. Seems like the KitsRoot is correct, at least there is a Lib folder in `C:\Program Files (x86)\Windows Kits\8.1\`.